src/en/pages/index.md

```diff
@@ -4,6 +4,87 @@ layout: "layouts/base.njk"
 permalink: /en/
 translationKey: "index"
 redirect_from: /
+onThisPage:
```

Comment:
Just a quick question. I know we made this menu specifically for use on documentation pages. What's the reasoning behind also adding it to the home page?

Reply:
I was gonna ask if there's a way of hiding the menu but keeping that column blank?

Reply:
also noticed, that anchor links are being generated automatically on the headings, with a hover state, kinda confusing wthout the side menu.

Reply:
Ethan and I spoke on slack and he advised me the best way to do this would be to create a new page layout. 

I'm just going to create a new one that limits character length for the content to about 80-85ch.